src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -142,10 +142,11 @@ DlgPrefLibrary::~DlgPrefLibrary() = default;
 
 void DlgPrefLibrary::slotShow() {
     m_bAddedDirectory = false;
+    m_bRelocatedDir = false;
```

Comment:
Like `m_bAddedDirectory` this should also be initializedin the constructor, not only on show